webpack/assets/javascripts/bundle_multiselect.js

```diff
@@ -0,0 +1,2 @@
+require('multiselect');
```

Comment:
looks like hound uses an older version of eslint that doesn't support this, what do you think about removing this rule @gailsteiger ?